apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-estimates.tsx

```diff
@@ -71,7 +131,21 @@ export const DealTicketEstimates = ({
           &nbsp;
           <small>({quoteName})</small>
         </dt>
-        <dd>{estCloseOut}</dd>
+        <dd className="flex gap-x-5 items-center">
```

Comment:
Chore (non-blocking): There's a lot of repetition here, it might be nice to pull these dds into a functional component and pass it the text

Reply:
yeah, good point, agree, changed :+1: